sonar-project.properties

```
@@ -0,0 +1,17 @@
+sonar.projectKey=sap_cloud-commerce-spartacus-storefront
+
+sonar.projectName=Cloud commerce spartacus storefront
+sonar.projectVersion=0.1.0
+
+sonar.sources=projects/storefrontlib,projects/core,projects/storefrontstyles
+sonar.exclusions=**/node_modules/**,**/*.spec.ts
+
+sonar.tests=projects/storefrontlib,projects/core,projects/storefrontstyles,projects/storefrontapp-e2e
+sonar.test.inclusions=**/*.spec.ts
```

E2E tests have different extension `**/*.e2e-spec.ts`
I assume that if we include directory with them, we also want to check them.
```suggestion
sonar.test.inclusions=**/*.spec.ts,**/*.e2e-spec.ts
```

---

After looking into it more it seems to work correctly without it.